mama/c_cpp/src/c/subscription.c

```diff
@@ -1550,31 +1560,31 @@ mama_status mamaSubscription_deactivate_internal(mamaSubscriptionImpl *impl)
 
 void MAMACALLTYPE mamaSubscription_DestroyThroughQueueCB(mamaQueue Queue, void* closure)
```

Comment:
Please revert all changes in this file from here on down (again, unless I have missed some functional change somewhere which is relevant to publisher events.